test/test_fx.py

```diff
@@ -2258,6 +2259,218 @@ def test_get_torch_func_signature_exhaustive(self, device, dtype, op):
+        "grid_sample": CONTROL_FLOW,
+        "gumbel_softmax": CONTROL_FLOW,
+        "hardsigmoid": CONTROL_FLOW,
+        "hardswish": CONTROL_FLOW,
+        "hardtanh": CONTROL_FLOW,
+        'hinge_embedding_loss': CONTROL_FLOW,
+        'huber_loss': CONTROL_FLOW,
+        'interpolate': CONTROL_FLOW,
+        'kl_div': CONTROL_FLOW,
+        'l1_loss': CONTROL_FLOW,
+        'leaky_relu': CONTROL_FLOW,
+        'local_response_norm': CONTROL_FLOW,
+        'margin_ranking_loss': CONTROL_FLOW,
+        'mse_loss': CONTROL_FLOW,
+        'multi_head_attention_forward': CONTROL_FLOW,
+        'multi_margin_loss': CONTROL_FLOW,
+        'multilabel_margin_loss': CONTROL_FLOW,
+        'multilabel_soft_margin_loss': CONTROL_FLOW,
+        'nll_loss': CONTROL_FLOW,
+        'poisson_nll_loss': CONTROL_FLOW,
+        'relu': CONTROL_FLOW,
+        'relu6': CONTROL_FLOW,
+        'rrelu': CONTROL_FLOW,
+        'selu': CONTROL_FLOW,
+        'silu': CONTROL_FLOW,
+        'smooth_l1_loss': CONTROL_FLOW,
+        'soft_margin_loss': CONTROL_FLOW,
+        'threshold': CONTROL_FLOW,
+        'triplet_margin_loss': CONTROL_FLOW,
+        'triplet_margin_with_distance_loss': CONTROL_FLOW,
+        'unfold': CONTROL_FLOW,
+        'upsample': CONTROL_FLOW,
+
+        "upsample_bilinear": INTERPOLATE_ARGS_CONFLICT,
+        "upsample_nearest": INTERPOLATE_ARGS_CONFLICT,
+    }
+
+    # List of nn.functionals with Tensor inputs but not with type annotation
+    FUNCTIONALS_WITHOUT_ANNOTATION = (
+        "adaptive_max_pool1d",
+        "adaptive_max_pool2d",
+        "adaptive_max_pool3d",
+        "fractional_max_pool2d",
+        "fractional_max_pool3d",
+        "max_pool1d",
+        "max_pool2d",
+        "max_pool3d",
+        "gaussian_nll_loss",
+        "upsample",
+        "upsample_bilinear",
+        "upsample_nearest",
+    )
+
+    def _test_nn_functional(self, func_name, fn):
+        if func_name in self.UNTRACEABLE_FUNCTIONALS:
+            exc, err = self.UNTRACEABLE_FUNCTIONALS[func_name]
+            with self.assertRaisesRegex(exc, err):
+                symbolic_trace(fn)
+            # Remove from UNTRACEBLE
+            del self.UNTRACEABLE_FUNCTIONALS[func_name]
+        else:
+            symbolic_trace(fn)
+
+    def _get_functional(self):
+        functional_list = []
+        for f in dir(torch.nn.functional):
+            if not f.islower():
+                continue
+            # Ignore internal functions
+            if f.startswith('_'):
+                continue
+            # Ignore supporting functions
+            if f in self.IGNORE_FUNCS:
+                continue
+            fn = getattr(torch.nn.functional, f)
+            # Ignore non-callable object like modules
+            if not isinstance(fn, Callable):
+                continue
+            # Remove built-in function/method
+            #  if not hasattr(fn, '__module__') or fn.__module__ != 'torch.nn.functional':
+            #      continue
```

Comment:
Like `adaptive_avg_pool1d`, it's bound to `torch.adaptive_avg_pool1d`, which is not a functional. We can ignore them can remove the `BUILT_IN_FUNC` from expected error.

Reply:
Is this commented check still needed? Seems like this case is covered by BUILT_IN_FUNC exception checking